rest/importtest/import_test.go

```diff
@@ -2378,3 +2379,75 @@ func TestImportUpdateExpiry(t *testing.T) {
 		})
 	}
 }
+
+func TestPrevRevNoPopulationImportFeed(t *testing.T) {
+	base.SkipImportTestsIfNotEnabled(t)
+	if base.UnitTestUrlIsWalrus() {
+		t.Skipf("test requires CBS for previous rev no assertion")
+	}
+
+	rtConfig := rest.RestTesterConfig{
+		DatabaseConfig: &rest.DatabaseConfig{DbConfig: rest.DbConfig{
+			AutoImport: true,
+		}},
+	}
+
+	rt := rest.NewRestTester(t, &rtConfig)
+	defer rt.Close()
+	dataStore := rt.GetSingleDataStore()
+	ctx := base.TestCtx(t)
+
+	if !rt.Bucket().IsSupported(sgbucket.BucketStoreFeatureMultiXattrSubdocOperations) {
+		t.Skip("Test requires multi-xattr subdoc operations, CBS 7.6 or higher")
+	}
+
+	// Create doc via the SDK
+	mobileKey := t.Name()
+	mobileBody := make(map[string]interface{})
+	mobileBody["channels"] = "ABC"
+	_, err := dataStore.Add(mobileKey, 0, mobileBody)
+	assert.NoError(t, err, "Error writing SDK doc")
+
+	// Wait for import
+	err = rt.WaitForCondition(func() bool {
+		return rt.GetDatabase().DbStats.SharedBucketImportStats.ImportCount.Value() == 1
+	})
+	require.NoError(t, err)
+
+	xattrs, _, err := dataStore.GetXattrs(ctx, mobileKey, []string{base.MouXattrName})
+	require.NoError(t, err)
+	if rt.GetDatabase().UseMou() {
+		var mou *db.MetadataOnlyUpdate
+		mouXattr, ok := xattrs[base.MouXattrName]
+		require.True(t, ok)
+		require.NoError(t, base.JSONUnmarshal(mouXattr, &mou))
+		// curr rev no should be 2, so prev rev is 1
+		assert.Equal(t, uint64(1), mou.PreviousRevID)
```

Comment:
I am really surprised the value of this is 1 at all, I would have expected that the value would be a higher number because my understanding was that this represents all documents in a bucket, and we have all the config documents in a bucket.

Additionally, I don't think that this test will work correctly when run with other tests at the same time, becuase we do not flush the bucket between tests, so other tests will pick up the revseqno first?

I think it might be correct to write the document _before_ the database is created, and get xattrs from `$document.revid`?